frontend/src/lib/widgetTheme.ts

```diff
@@ -153,28 +158,30 @@ export const fileUploaderOverrides: FileUploaderOverrides<StyleProps> = {
       paddingLeft: "0.25rem",
       paddingRight: "0.25rem",
       height: "4.25rem",
-      borderColor: $isDragActive ? primary : "transparent",
-      backgroundColor: $isDragActive ? primaryA50 : $theme.colors.mono200,
+      borderColor: $isDragActive ? colors.primary : "transparent",
```

Comment:
Now I'm wondering if "transparent" counts as a color -- aka should we include it inside our `colors` group? (No strong opinion either way.)

Reply:
Doesn't hurt, will update!